plugins/woocommerce/src/Internal/DataStores/Orders/OrdersTableDataStore.php

```diff
@@ -539,7 +539,28 @@ public function get_stock_reduced( $order ) {
 	public function set_stock_reduced( $order, $set, $save = true ) {
 		// XXX implement $save = true.
 		$order = is_numeric( $order ) ? wc_get_order( $order ) : $order;
-		return $order->update_meta_data( '_order_stock_reduced', wc_string_to_bool( $set ) );
+		return $order->update_meta_data( '_order_stock_reduced', wc_bool_to_string( $set ) );
```

Comment:
before launch, we should come back here and add logic to these small update in bulk, otherwise we are still doing lot of queries in inserting just one order.

Reply:
Yes. We definitely need to go through a bunch of TODO's and XXX's before launch. The blocker I added to the P2 post should remind us of these pending tasks, though I wouldn't mind creating a specific issue here in GH for that if you prefer that.

Reply:
The blocker list is fine, I go through it from time to time, so we wouldn't miss it.